components/centraldashboard/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+# build stage
+FROM golang:alpine AS build-env
+ADD . /go/src/github.com/google/kubeflow
+RUN go install github.com/google/kubeflow/components/centraldashboard/backend
+RUN mv /go/bin/backend /go/bin/centraldashboard
+# final stage
+FROM alpine
+WORKDIR /app
+COPY --from=build-env /go/bin/centraldashboard /app/
+ENTRYPOINT ./centraldashboard
```

Comment:
nit: No newline at EOF.

Reply:
No. That is not the official logo. However, I used the same logo that was there in the k8s dashboard for continuity's sake. 

![logo](https://raw.githubusercontent.com/tensorflow/k8s/ed7cae7552bacfa4d693fb8b5975d2e1ec62194a/dashboard/frontend/public/favicon.ico)